1-BasicSetUp.sh

```diff
@@ -57,7 +57,11 @@ cp ./config_files/.tmux.conf ~
 cp ./config_files/.tmux.conf.local ~
 mkdir -p ~/.config/tilda
 cp ./config_files/config_0 ~/.config/tilda/
-
```

Comment:
Keep the empty line, makes it easier to read the script

Reply:
Extra Line Added before and after the Deletion Part.